src/atom-config/make-handler/index.js

```diff
@@ -0,0 +1,9 @@
+'use babel'
+
+/* eslint-disable no-param-reassign */
```

Comment:
Never blanket disable a rule without an amazingly good reason.

I don't even see where this has a purpose in this file!

Reply:
That is a stray from a point in time when the linked exports were all inside the file. 🔥

Reply:
see reply under `lint-html.js`